dist/common/scripts/scylla-housekeeping

```diff
@@ -96,11 +96,9 @@ def sanitize_version(version):
     false negative version_compare() checks.
     """
```

Comment:
> This reverts commit https://github.com/scylladb/scylladb/commit/65fbf72ed0f0980381c3307cda5a6d1b1fde2dfb.

Please add an explanation here why it's reverted.

Reply:
Added an explanation on revert commit.

Reply:
I don't see the explanation on this commit. It still says "This reverts commit https://github.com/scylladb/scylladb/commit/65fbf72ed0f0980381c3307cda5a6d1b1fde2dfb.".

Reply:
Sorry I mistakenly did not pushed it.
Pushed it now:
https://github.com/scylladb/scylladb/pull/19670/commits/db04f8b16e4f5e5fcefaf67523a501948bca16ba